src/tools/pub/package-layout.md

```diff
@@ -217,8 +219,9 @@ For more information on library packages, see
 
 Dart scripts placed inside of the `bin` directory are public. Any package
 that depends on your package can run scripts from your package's `bin`
-directory using [`pub run`](/tools/pub/cmd/pub-run). <em>Any</em> package can run scripts
-from your package's bin directory using [`pub global`](/tools/pub/cmd/pub-global).
+directory using [`pub run`](/tools/pub/cmd/pub-run). <em>Any</em> package
```

Comment:
I don't think this is your change, but I never thought about packages running scripts (although of course they can)—I think of _people_ running scripts. I guess we don't have to cover this here, though.